File: 0771-jewels-and-stones/0771-jewels-and-stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
      int no=0;
        for(int i=0;i<stones.length();i++){
            for(int j=0;j<jewels.length();j++){
                if(jewels[j]==stones[i]){
                    no++;     
                }

            }
        }
        return no;
    }
};

File: 0771-jewels-and-stones/0771-jewels-and-stones.py
int numJewelsInStones(char * jewels, char * stones){
   int num=0;
   for(int i=0;i<strlen(stones);i++){
       for(int j=0;j<strlen(jewels);j++){
           if(stones[i]==jewels[j]){
               num++;
           }
       }
   }
return num;
}